modules/openy_gc_auth/modules/openy_gc_auth_custom/openy_gc_auth_custom.info.yml

```diff
@@ -7,8 +7,8 @@ core: 8.x
 core_version_requirement: ^8 || ^9
 dependencies:
   - drupal:migrate
-  - drupal:migrate_source_csv
-  - drupal:migrate_plus
+  - drupal:migrate_source_csv (>=8.x-3.4)
+  - drupal:migrate_plus (>=8.x-5.1)
```

Comment:
Please, add also the `migrate_tools` dependency

Reply:
fixed